.bit/tests/test.1.8.js

```diff
@@ -34,6 +34,28 @@ try {
             console.log(`YIKES! We got ${data} instead of ${answer}. Try again!`)
             process.exit(1)
         }
+
+        try {
+            const resp2 = await fetch(uri, {
+                method: 'GET'
+            });
+            var data2 = await resp2.text()
+            let test2 = JSON.stringify(data2)
+            functions.getStatus(resp2, uri)
+        } catch (e) {
+            console.log("We're having trouble making a request to your endpoint when plaintext is blank. Try again?")
+            process.exit(1)
+        }
+
+        if (data2.length < 3) {
+            console.log("No response... Try again!")
+            process.exit(1)
+        } else if (data2 == "Please enter some text to convert!") {
+            console.log("Also, great work catching a blank plaintext parameter value.")
+        } else {
+            console.log(`Sorry! You forgot to check for a blank plaintext. If we sending nothing in "plaintext," we should get "Please enter some text to convert!" Instead, we got ${data2}`)
```

Comment:
> Some times we have a log followed by exit process, other times throw new Error. I guess the log approach helps as you don't always know if there is a try-catch. That said it would be worth seeing how git hub handles the two scenarios and which looks better. If you are to keep with log and process exist, I would suggest using console.Error which does the same as log but its text in red. Console.Info is also useful for other text.

I will modify that in all the tests to make this consistent. Using throw new error produces a lot of unneeded text, so I will do:
```js
console.error()
process.exit(1)
```

```js
console.info()
```